workspaces/spectacle-shared/src/local-cli/client.ts

```diff
@@ -22,7 +22,10 @@ import {
 import { IApiCliConfig } from '@useoptic/cli-config';
 
 export class LocalCliSpectacle implements IForkableSpectacle {
-  constructor(private baseUrl: string, private opticEngine: IOpticEngine) {}
+  private spectacleUpdateEvents: ((...args: any) => any)[];
+  constructor(private baseUrl: string, private opticEngine: IOpticEngine) {
+    this.spectacleUpdateEvents = [];
```

Comment:
We need to implement this because:
- The local cli version doesn't have a spectacle watcher on the UI side
- only the in memory spectacle instances have a spec change notification watcher

We force an identity change in spectacle (which is the same as how the in memory instances work) whenever we call a mutation. Since we aren't actually making a web socket connection to our cli-server, we listen to when we send a mutate event. 

We can tackle this in a separate PR, but do we want to create a web-socket connection between the cli-server and the UI? Would that make sense, or would we expect users to refresh their page if they switched branches, or something changed on the file system outside of optic? (i.e. connecting this https://github.com/opticdev/optic/pull/826 to the UI)